endpoints/api/namespacequota.py

```diff
@@ -76,7 +76,7 @@ class OrganizationQuotaList(ApiResource):
     @nickname("listOrganizationQuota")
     def get(self, orgname):
         orgperm = OrganizationMemberPermission(orgname)
-        if not orgperm.can():
+        if not orgperm.can() and not SuperUserPermission().can():
```

Comment:
this endpoint wont be called by super users it should now call the super users endpoint.

Reply:
The super users endpoint only have `put` and `delete` methods as [here](https://github.com/quay/quay/blob/master/endpoints/api/superuser.py#L274)